app/assets/stylesheets/molecules/_toolbar.scss

```
@@ -1,36 +1,44 @@
+$toolbar-item-wrap-spacing: .2em;
```

```suggestion
$toolbar-item-wrap-spacing: .2em !default;
```

...would make this more extendable.

---

Good call, done.